docs/index.fsx

```diff
@@ -0,0 +1,306 @@
+(*** hide ***)
+/// example fast binder url: https://mybinder.org/v2/gh/fsprojects/fsharp.formatting/master?urlpath=git-pull?repo=https:/nhirschey.github.com/teaching/gh-pages/fundamentals.ipynb
+
+(**
+
+[![Binder](img/badge-binder.svg)](https://mybinder.org/v2/gh/fsprojects/FSharp.Data.Fred/gh-pages?filepath={{fsdocs-source-basename}}.ipynb)&emsp;
+[![Script](img/badge-script.svg)]({{root}}/{{fsdocs-source-basename}}.fsx)&emsp;
+[![Notebook](img/badge-notebook.svg)]({{root}}/{{fsdocs-source-basename}}.ipynb)
+*)
+
+(*** hide ***)
+#r "../src/FSharp.Data.Fred/bin/Debug/net5.0/FSharp.Data.Fred.dll"
+open FSharp.Data.Fred
+
+(**
+F# Data FRED
+===================
+
+F# Data FRED is a library for
+[FRED database](https://fred.stlouisfed.org/) data access
+based on FSharp.Data. 
+
+Short for Federal Reserve Economic Data, FRED is an online database consisting 
+of hundreds of thousands of economic data time series from scores of national, 
+international, public, and private sources.
+
+This library uses the [FRED API](https://fred.stlouisfed.org/docs/api/fred/) to access the data.
+*)
+
+(**
+You can use `FSharp.Data.Fred` in [dotnet interactive](https://github.com/dotnet/interactive) 
+notebooks in [Visual Studio Code](https://code.visualstudio.com/) 
+or [Jupyter](https://jupyter.org/), or in F# scripts (`.fsx` files), 
+by referencing the package as follows:
+
+    // Use one of the following two lines
+    #r "nuget: FSharp.Data.Fred" // Use the latest version
+    #r "nuget: FSharp.Data.Fred,{{fsdocs-package-version}}" // Use a specific version   
+*)
+
+#r "nuget: FSharp.Data" //Also load FSharp.Data
+
+open FSharp.Data
+open FSharp.Data.Fred
+open System //Usefull to access DateTime
+
+(**
+## F# Data FRED
+First in order to use the methods you'll need to create a value with type `Fred`. 
+
+Fred receives an API key as a parameter (`string`). [Get you FRED API key.](https://fred.stlouisfed.org/docs/api/api_key.html)
+
+Example: 
+
+    let apiKey = "insert API key here"
+*)
+
+(*** hide ***)
+let apiKey = "b1ff5229658ed54b63929049b5feca91"
```

Comment:
It would be nice to not hard code your apiKey into the documentation. What about adding a single file called `fredKey.json` to .gitignore and then create two files at the root directory:

1. a file called `fredKey.json`, this will be ignored by git.
2. a file called `sample.fredKey.json`, this will be committed to git.

The sample file should look like
```fsharp
{ "fredKey": "key here in the actual file that you should name fredKey.json and NOT commit to git"}
```
Then also create your real key file with that format (but remember it won't be committed to git).

In JsonApi.fs you'll then want to create a type for that file:

```fsharp
type KeyFile = JsonProvider<""" { "fredKey": "key here in the actual file to be named fredKey.json"} """>
```
Then in documenation code here that you hiding instead of the hard coded key you can do something like

```fsharp
    let envVars = System.Environment.GetEnvironmentVariables() 
    let apiKey = 
        let var = "FRED_KEY"
        let keyJson = "../fredkey.json"
        if envVars.Contains var then 
            envVars.[var] :?> string
        elif File.Exists(keyJson) then 
            KeyFile.Parse(keyJson).fredKey
        else failwith "could not find a key"
```

The environment variables will be set as github secrets for workflow actions. I'm typing this on the web so the above code might have typos. This is similar to what I did [here](https://github.com/nhirschey/Teaching/blob/440fbd68e6463a98e246cc43d8843176fb507abb/docs/Common.fsx#L13)